package cmexample1domain.databasevalues;

message FooDescription {
    optional string description = 1;
}
